File: app/src/main/java/com/example/massa/luxvilla/utils/NetworkCheck.java
package com.example.massa.luxvilla.utils;

import android.content.Context;
import android.net.ConnectivityManager;

/**
 * Created by massa on 30/03/2017.
 */

public class NetworkCheck {
    public static boolean isNetworkAvailable(final Context context) {
        final ConnectivityManager connectivityManager = ((ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE));
        return (connectivityManager != null ? connectivityManager.getActiveNetworkInfo() : null) != null && connectivityManager.getActiveNetworkInfo().isConnected();
    }
}

File: app/src/main/java/com/example/massa/luxvilla/utils/NetworkCheck.kt
package com.example.massa.luxvilla.utils

import android.content.Context
import android.net.ConnectivityManager

/**
 * Created by massa on 27/10/2017.
 */
object NetworkCheck {
    fun isNetworkAvailable(context: Context): Boolean {
        val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        return connectivityManager.activeNetworkInfo != null && connectivityManager.activeNetworkInfo.isConnected
    }
}